akka-persistence/src/main/scala/akka/persistence/journal/WriteJournalBase.scala

```diff
@@ -13,8 +13,8 @@ import akka.persistence.AtomicWrite
 private[akka] trait WriteJournalBase {
   this: Actor ⇒
 
-  lazy val persistence = Persistence(context.system)
-  private def eventAdapters = persistence.adaptersFor(self)
+  val persistence = Persistence(context.system)
+  private val eventAdapters = persistence.adaptersFor(self)
```

Comment:
Though that may be a non-issue, since we discussed last time that the ad-hoc have limited usability and we'll most likely not support that...